torch/_inductor/codegen/cpp.py

```diff
@@ -4905,23 +4913,14 @@ def template_buffer_has_other_users(
             flag_template_buffer_has_other_users=flag_template_buffer_has_other_users,
             epilogue_nodes=epilogue_ir_nodes,
         )
-        with kernel:
-            if isinstance(template_node.node, ir.CppTemplateBuffer) and isinstance(
+
+        def _is_grouped_gemm(template_node):
```

Comment:
Is this specific to "grouped gemm" or just about template with multiple outputs?

Reply:
Thanks for the comment. Change the name to `_is_multi_outputs_template`.